bower.json

```diff
@@ -29,7 +29,7 @@
     "px-actionable-design": "^2.0.5",
     "px-popover": "^2.1.2",
     "px-slider": "^2.1.1",
-    "px-rangepicker": "^3.2.0",
+    "px-rangepicker": "v3.3.4",
```

Comment:
why was `^` changed to `v`? That means we are stuck with the particular version and won't get any bug-fixes.

Reply:
My mistake, fixing before merging. Good catch.